bigquery/tests/unit/test_client.py

```diff
@@ -1113,19 +1113,57 @@ def test_copy_table(self):
         DATASET = 'dataset_name'
         SOURCE = 'source_table'
         DESTINATION = 'destination_table'
+        RESOURCE = {
+            'jobReference': {
+                'projectId': PROJECT,
+                'jobId': JOB,
+            },
+            'configuration': {
+                'copy': {
+                    'sourceTable': {
+                        'projectId': PROJECT,
+                        'datasetId': DATASET,
+                        'tableId': SOURCE,
+                    },
+                    'destinationTable': {
+                        'projectId': PROJECT,
+                        'datasetId': DATASET,
+                        'tableId': DESTINATION,
+                    },
+                },
+            },
+        }
         creds = _make_credentials()
         http = object()
         client = self._make_one(project=PROJECT, credentials=creds, _http=http)
+        conn = client._connection = _Connection(RESOURCE)
         dataset = client.dataset(DATASET)
         source = dataset.table(SOURCE)
         destination = dataset.table(DESTINATION)
-        job = client.copy_table(JOB, destination, source)
+
+        job = client.copy_table(source, destination, job_id=JOB)
+
+        # Check that copy_table actually starts the job.
+        self.assertEqual(len(conn._requested), 1)
+        req = conn._requested[0]
+        self.assertEqual(req['method'], 'POST')
+        self.assertEqual(req['path'], '/projects/PROJECT/jobs')
```

Comment:
This seems correct by "accident". It chould be `'/projects/%s/jobs' % PROJECT`, I believe.

Reply:
Good catch. Fixed.